src/js/view/view.js

```diff
@@ -417,12 +417,15 @@ define([
 
         function itemReady(item) {
             const provider = _model.getVideo();
-            if (provider && provider.getName().name.indexOf('flash') === 0) {
+            // Youtube, chromecast and flash providers do no support video tags
+            if (provider && /^(youtube|chromecast|flash)/.test(provider.getName().name)) {
                 return;
             }
-            const title = item.title || '';
             var videotag = _videoLayer.querySelector('video, audio');
-            videotag.setAttribute('title', title);
+            const dummyDiv = document.createElement('DIV');
+            // Writing a string to innerHTML completely decodes multiple-encoded strings
+            dummyDiv.innerHTML = item.title || '';
+            videotag.setAttribute('title', dummyDiv.innerHTML);
```

Comment:
@robwalch @karimJWP `textContent` does not decode html encoded strings. Also, setting `innerHTML` then reading `innerText` ensures <, >, ", ' and ` get decoded correctly.